ckanext/cfpb_extrafields/templates/package/search.html

```diff
@@ -1,9 +1,10 @@
 {% ckan_extends %}
-{% block breadcrumb_content%}
+{% block breadcrumb %}
 {{ super() }}
-<span style="padding-left:450px;float:right;overflow:hidden">
+<span style="float:right;overflow:hidden">
```

Comment:
@mattboehm If I remove the padding-left completely, the "Add Dataset" link is right up against the breadcrumb links, maybe keep 10px of padding to give it a little separation?  Also, the float can be removed completely for now, since it won't work as desired with a "span" element.